database/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM postgres:9.6.2-alpine
+
+RUN apk update \
+&& apk upgrade \
+&& apk add -q alpine-sdk \
+&& apk add -q vim \
+&& apk add -q nano;
+
```

Comment:
Oh absolutely.

If you do `docker exec -ti db /bin/sh`

That will log you in as root.